app/main/posts/views/mode-context-form-filter.directive.js

```diff
@@ -116,18 +116,26 @@ function ModeContextFormFilter($scope, FormEndpoint, PostEndpoint, TagEndpoint,
             include_unmapped: true
         });
 
+        // @uirouter-refactor - Ask Anna??
```

Comment:
@Angamanga here

Reply:
to reproduce the erorr I was having, go to sandbox and qa in different tabs/browsers.

Select a saved search. 
See that the sidebar/modecontextbar updates correctly in Sandbox but in QA it doesn't take the corrent count of posts by each survey . I noticed this was happening with categories at least.